dynamicprefixvici.c

```diff
@@ -157,19 +234,180 @@ static void ParseCommandLineArguments(int argc, char** argv, CommandLineArgument
         }
     }
 
-    // Check whether a pool name and a prefix is defined
-    if(arguments->pool_name == NULL && arguments->prefix == NULL)
+    // Check whether a pool name and a prefix_address is defined
```

Comment:
Kunnen we deze extra argumenten checks niet onderbrengen in de functie
CheckValidityOfArguments, dan hebben we éentje voor de parser en ééntje voor de checks logisch bij elkaar.